package.json

```diff
@@ -18,6 +18,7 @@
     "docs-install": "npm run _cd:docs -- npm install",
     "fix:format": "npm run _check:format -- --write",
     "get:submodule": "set -x && git submodule update --init ${DEPTH:- --depth 1}",
+    "postinstall": "cd .. && npx mkdirp github.com/FortAwesome/Font-Awesome && npx mkdirp github.com/twbs/bootstrap",
```

Comment:
```suggestion
    "postinstall": "npx -no mkdirp ../github.com/FortAwesome/Font-Awesome ../github.com/twbs/bootstrap",
```

Reply:
Do you mean npx `--no` (two dashes)?

Reply:
Oops, yes. Fixed.

Reply:
Resolved with commit 396d76d5358019fa7bb4a5146bb7811ef7804401 (as suggested).